src/main/java/io/fabric8/maven/docker/SaveMojo.java

```diff
@@ -106,7 +106,7 @@ private String completeCalculatedFileName(String file) throws MojoExecutionExcep
     }
 
     private void ensureSaveDir(String fileName) throws MojoExecutionException {
-        File saveDir = new File(fileName).getParentFile();
+        File saveDir = new File(fileName).getAbsoluteFile().getParentFile();
```

Comment:
thanks ! @rohanKanojia : FYI, this fix very likely fixes also #1200 if applied to `getContextDir()`

Reply:
okay, thanks